src/lib/yang/binary.lua

```diff
@@ -434,12 +437,30 @@ function selftest()
 
       leaf-list integers { type uint32; }
       leaf-list addrs { type inet:ipv4-address; }
+
+      typedef severity  {
+         type enumeration {
+            enum indetermiante {
+               value 2;
+            }
+            enum minor {
```

Comment:
Can you leave off the value for one of these?  Just to make sure we are not requiring that the value be present.